job-service/src/main/java/com/hpe/caf/services/job/api/filter/JdbcQueryBuilder.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016-2020 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.services.job.api.filter;
+
+import com.healthmarketscience.sqlbuilder.ComboCondition;
+import cz.jirutka.rsql.parser.ast.AndNode;
+import cz.jirutka.rsql.parser.ast.ComparisonNode;
+import cz.jirutka.rsql.parser.ast.LogicalNode;
+import cz.jirutka.rsql.parser.ast.Node;
+import cz.jirutka.rsql.parser.ast.OrNode;
+import com.healthmarketscience.sqlbuilder.Condition;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+public class JdbcQueryBuilder
+{
+    public Condition createQuery(final ComparisonNode comparisonNode)
+    {
+        final Condition result = JobFilterQuery.getQueryStatement(comparisonNode.getSelector(),
+                                                                  comparisonNode.getOperator(),
+                                                                  comparisonNode.getArguments());
+        return result;
+    }
+
+    public Condition createQuery(final AndNode logicalNode)
+    {
+        return getStream(logicalNode).reduce((r, s) -> ComboCondition.and(r, s)).get();

```

Comment:
Pointless lambda just passing the parameters it was given.

```suggestion
        return getStream(logicalNode).reduce(ComboCondition::and).get();
```